### PR TITLE
Changed Background

### PR DESCRIPTION
**Pull request by Kendallfuller97**
#Changes
List Changes Introduced by this PR
*Item 1
*Item 2

#Purpose
Describe the problem or feature.

#Approach
How does this change address the problem?

#Learning
Describe the research stage
Links to blog posts, patterns, libraries or addons used to solve this problem

#If this closes an issue, reference the issue here. If it doesn't, remove this line 
Closes #000

### CODE REVIEW

**Review by Eddie-Moreno — APPROVED** [review]
Looks good!

**Review by jacquelinemartinez6114 — APPROVED** [review]
Background color changed to background-color: #00c380;

**Review by Eddie-Moreno — APPROVED** [review]
Looks good!

File: Kenny/Kenny.html [hunk 1]
```diff
@@ -6,9 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Kendall Fuller</title>
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="..">
+    <link rel="stylesheet" href="Kenny.css">
+    <script src="Kenny.js" defer ></script>
```
Comment (gabymoreno9) [inline Kenny/Kenny.html:12]: Example: great use of external script

**Pull request merged**